docs/features/anti-affinity/anti-affinity.md

```diff
@@ -0,0 +1,60 @@
+# Anti Affinity
+
+Depending on a cluster's configuration, a Blue Green Rollout (or a Canary rollout that uses traffic management) can cause newly created pods to restart after deploying a new version. This can be problematic, especially for applications that cannot startup quickly or do not gracefully exit.
+
+This behavior occurs because cluster auto-scaler wants to scale down the extra capacity which was created to support a Rollout
+running in double capacity. It usually happens if a Rollout has its own dedicated instance group since a Rollout has a greater effect on
```

Comment:
We should add a quick comment connecting this back to the restarting pods. I.E. If the scaled-down node had a pod owned the Rollout, that pod would be deleted and recreated.